vitest.config.js

```diff
@@ -0,0 +1,13 @@
+import { defineConfig } from 'vitest/config'
+
+export default defineConfig({
+  test: {
+    environment: 'jsdom',
+    environmentOptions: {
+      jsdom: {
+        customExportConditions: ['node', 'node-addons']
+      }
+    },
+    globals: true
```

Comment:
this allows us to use vitests test functions like (`beforeEach`, `afterEach`, `describe`, `test`, etc.) without importing them in each test file.